workflow/realisations.py

```diff
@@ -363,6 +384,36 @@ class SRFConfig(RealisationConfiguration):
     resolution: float
     """The resolution of the SRF geometry"""
 
+    point_source_params: PointSourceParams | None
```

Comment:
Move `PointSourceParams` class to schema
```suggestion
    point_source_params: schemas.PointSourceParams | None
```